### DISCUSSION TITLE
Local PostCSS configuration is not applied to CSS files imported from node_modules when using workspaces

### DISCUSSION THREAD

**Discussion by realduolaf** [vercel/next.js#54324] [Help]
### Summary

~~When I tried to migrate to the app router, postcss.config.js did not work.~~

~~Is it no longer supported postcss with app router?~~

https://nextjs.org/docs/app/building-your-application/configuring
https://nextjs.org/docs/app/building-your-application/styling

Neither link mentions postcss.

I just found only not work when css imported from node_modules.

need help.

**Comment by icyJoseph** [comment]
It is supported, if you create a `create-next-app` with App Router and Tailwind, you'll see a `postcss.config.js` file.

**Reply by realduolaf** [reply]
I'm using [turboRepo](https://turbo.build/repo) and I just found an https://github.com/vercel/turbo/issues/5068. related to mine.
The issue mentions that this issue has been fixed in version 13.4.3. But not work for me. The version of next I use is 13.4.16. Still a problem.
